controllers/openstackbaremetalset_controller.go

```diff
@@ -436,10 +529,18 @@ func (r *OpenStackBaremetalSetReconciler) ensureBaremetalHosts(instance *ospdire
 		for i := 0; i < oldBmhsToRemoveCount; i++ {
 			// First choose BaremetalHosts to remove from the prepared list of BaremetalHosts
 			// that have the "osp-director.openstack.org/baremetalset-delete-baremetalhost=yes" annotation
```

Comment:
I must have missed this last review, but we should probably change the annotation comment here so it's not referring to the old annotation (just to avoid potentially confusing people)

Reply:
sure, good catch. found it also in another place

Reply:
done